src/main/java/StringCalculatorStep1.java

```diff
@@ -5,7 +5,8 @@ public static final int Add(final String numbers) {
         int returnValue = 0;
         String[] numbersArray = numbers.split(",");
         if (numbersArray.length > 2) {
-            throw new RuntimeException("Until 2 numbers seperated by commas is allowed");
+            throw new RuntimeException("Up to 2 numbers separated by comma (,) are allowed");
+
         } else {
             if (!numbers.isEmpty()) {
                 returnValue = Arrays.stream(numbersArray).mapToInt(Integer::parseInt).sum();
```

Comment:
The condition '!numbers.isEmpty()' can be optimized by using 'numbersArray.length > 0', as 'split' on an empty string will return an array of a single empty string.